### DISCUSSION TITLE
Add tag based on user who requested?

### DISCUSSION THREAD

**Discussion by Robert-D** [seerr-team/seerr#351] [Q&A] [answered]
Looking to move from OMBI to Jellyseerr. Mother-in-law requests all kinds of content that my wife and I just don't watch and we really don't want to see this content on our Jellyfin experience. In OMBI there is a feature to add the username tag to each request. We then can filter out content with that tag in Jellyfin. (Their requests don't effect our Jellyfin experience). Is there a way to do this Jellyseerr, while also allowing my mother-in-laws request to be automatically approve?

Only way I see of doing this would be to setup multiple servers with the tag selected, then have to override the server used when approving each request.

**Comment by fallenbagel** [comment, marked as answer]
Automatic tag addition with username serverwise is in the works. 

Currently there are three ways you can achieve this. 
- You can have one server and when you approve the requests you can select the tag.
- Give mum advanced request perm and let her select tag while requesting. (Not recommended)
- Through a webhook script (see [seerr-route](https://github.com/Fallenbag/seerr-route) on how it can be done. Tho this one changes directory, you can use the jellyseerr api and modify script to auto add tags too).

**Reply by Robert-D** [reply]
I'm so happy to hear it's in the works! Thank you so much for for everything you've committed to this project! Looking forward to see where it goes.
